src/test/excessiveblock_test.cpp

```diff
@@ -229,6 +238,9 @@ BOOST_AUTO_TEST_CASE(check_generated_block_validator)
 
     str = MiningBlockSizeValidator(tmpMGB, (uint64_t *) 42, true);
     BOOST_CHECK(! str.empty());
+
+    maxGeneratedBlock = c_mgb;
+    excessiveBlockSize = c_ebs;
 }
```

Comment:
So it seems that you fix a the test failure with 2002cb8 ! 

this is great. 

I wonder if this fix would deserve a PR on it's own. 

What's your take? 

Paging @thezerg